pycoal/mineral.py

```diff
@@ -285,7 +286,7 @@ def avngDNN(image_file_name, classified_file_name, model_file_name,
 
 class MineralClassification:
 
-    def __init__(self, algorithm=SAM, **kwargs):
+    def __init__(self, '''algorithm=SAM,''' **kwargs):
```

Comment:
Why wrap in single quotes. If you are not using then simply remove. 
I do do think we want to retain this parameter however or else we introduce an API regression. Please remove the quotes.